src/codecoverage/FileInfo.java

```diff
@@ -20,31 +23,63 @@
 		
 	}
 	public FileInfo(){}
-	public void getReportOfTestedFunction(){
+	public void getReportOfTestedFunction() throws IOException{
 		int numberOfDeclaredFunctions= functions.size();
 		int numberOfFunctionCalls= callfunctionNames.size();
 		int numberOfMatchedFunctions=0;
-		int a= 0;
+		
 		for(int i=0; i<numberOfDeclaredFunctions; i++)
 		{
 			
 			for(int j=0;j<numberOfFunctionCalls;j++)
 			{
-				//System.out.println("declared= "+functions.get(i)+" called function= "+callfunctionNames.get(j));
+				
 				if(callfunctionNames.get(j).equals(functions.get(i)))
 				{
 				   numberOfMatchedFunctions ++;
 				   totalNumberOfMatchesInAllFiles++;
 				}
 			}
 		}
-		//System.out.println("counterSmith="+totalNumberOfMatchesInAllFiles);
+		int percentage= (numberOfMatchedFunctions*100)/ numberOfDeclaredFunctions;
 		
+	    File fc = new File("/Users/ten24user/git/codecoverage/htmlOutput.html");
+	   
+        BufferedWriter bw = new BufferedWriter(new FileWriter(fc,true));
+  //      bw.write("<table>");
+ 
+        bw.write("<tr>");
+        
+        bw.write("<td>"+file.getName()+"</td>");
+        bw.write("<td>"+Boolean.toString(testFile.exists())+"</td>");
+        bw.write("<td>"+Integer.toString(numberOfMatchedFunctions)+"</td>");
+        bw.write("<td>"+Integer.toString(numberOfDeclaredFunctions)+"</td>");
+        bw.write("<td>"+Integer.toString(numberOfFunctionCalls)+"</td>");
+        if(percentage<=100){
+        bw.write("<td>"+Integer.toString(percentage)+"</td>");
+        }
+        else{
+        	bw.write("<td>"+"100"+"</td>");
+        }
+        bw.write("</tr>");
+        bw.write("</table>");
```

Comment:
i think that you probably want to close the table at the end of the service. This may make multiple table close tags